pyprof/nvtx/nvmarker.py

```diff
@@ -400,6 +499,8 @@ def foo(args, name):
     cadena = {}
     cadena['mod'] = mod.__name__
     cadena['op'] = op
+    cadena['callid'] = idx
```

Comment:
This will always include this extra information in the cadena, even when disabled and empty?